exe/Daemon.hs

```diff
@@ -238,6 +239,28 @@ send id (Api.SendRequest expressions) = do
         Nothing -> throw $ MachineError id AckTimeout
 
 
+-- | Give the HTML specified by a machine (via defining `(get-html)`).
+frontend :: MachineId -> [Text] -> Daemon Api.HtmlText
+frontend id path = do
+    m <- ensureMachineLoaded id
+    bumpPolling id
+    case runIdentity $ interpret "[frontend]" "(get-html)" (machineState m) of
+        Left err -> throw $ MachineError id (InvalidInput err)
```

Comment:
Maybe the error can be more informative, for example to include a message that the machine might not have defined a front-end at all, or not according to this convention.

Reply:
Fixed